### DISCUSSION TITLE
Is there a way to disable everything except for the Softmute and chat functionality?

### DISCUSSION THREAD

**Discussion by xDull** [GriefPrevention/GriefPrevention#826] [Q&A] [answered]
We have a large anarchy server, trying to find a way to incorporate your plugins chat ‘soft mute’ and hidden message to prevent excessive profanity but not inform the player they’ve been muted. but we do not wish to give players claims etc.
Is there a way to do this?

**Comment by RoboMWM** [comment, marked as answer]
yes, you can disable claiming and other stuff in the config
